pages/design/index.mdx

```diff
@@ -1,3 +1,16 @@
-# Guidelines
+# Introduction
+
+Our goal is to create an accessible and approachable system that keeps user experience consistent across the breadth of GitHub products and services while also allowing flexibility for new ideas. These guidelines are created with *all* users in mind. Primer Design sets foundations for design principles, accessibility, layouts, and content. We also provide downloadable resources for multiple design tools.
+
+## Designed with accessibility first
+We created Primer Design so that GitHub can be used by everyone. Our guidelines have considerations for users with visual, hearing, cognitive, and mobile disabilities.
+
+
+## Responsiveness
+Primer Design includes guidelines for page layouts, hierarchy, and navigation on both web and mobile.
+
+
+## Tools that encourage collaboration
+We use [Figma](http://figma.com) as our primary tool of choice because of its ability to create a more transparent work environment. Figma supports GitHub's mission to help people work together from all of the world and contribute to amazing projects.
```

Comment:
- `https://www.figma.com/`
- I think that both sentences say the same thing. Maybe we can combine into one?